charts/victoria-metrics-k8s-stack/files/rules/generated/vmcluster.yaml

```diff
@@ -35,7 +35,8 @@ rules:
     sum(vm_data_size_bytes) by (job,instance,{{ $clusterLabel }}) /
     (
      sum(vm_free_disk_space_bytes) by (job,instance,{{ $clusterLabel }}) +
-     sum(vm_data_size_bytes) by (job,instance,{{ $clusterLabel }})
+     sum(vm_data_size_bytes) by (job,instance,{{ $clusterLabel }}) -
+     sum(label_value(flag{name="storage.minFreeDiskSpaceBytes"}, "value")) by (job,instance,{{ $clusterLabel }})
```

Comment:
when I was adding our own alert for this in the meantime, I saw that `vm_free_disk_space_limit_bytes` already exists for this flag 😍